src/js/plugins/tts/PageChunk.js

```diff
@@ -90,6 +91,15 @@ export default class PageChunk {
 
         return rects;
     }
+
+    /**
+     * Remove "dangling" hyphens from read aloud text to avoid TTS stuttering
+     * @param {string} text 
+     * @return {string}
+     */
+    static _removeDanglingHyphens(text) {
+        return text.replace(/-\s+/g, '');
```

Comment:
i ran this through regexr, it looks like it will only split out the last hyphen, even though there can/may be multiple hyphens

https://regexr.com/4qsgv

Reply:
As discussed, leaving as is; if we notice it occurring in the wild, we can fix it :+1: